diff --git a/cyberdrop_dl/managers/progress_manager.py b/cyberdrop_dl/managers/progress_manager.py
index 855fe10b7850..fb2ccffc939d 100644
--- a/cyberdrop_dl/managers/progress_manager.py
+++ b/cyberdrop_dl/managers/progress_manager.py
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import time
+from datetime import timedelta
 from dataclasses import field
 from typing import TYPE_CHECKING
 
@@ -16,7 +17,6 @@ from cyberdrop_dl.utils.utilities import log_with_color, log, log_spacer, parse_
 
 if TYPE_CHECKING:
     from cyberdrop_dl.managers.manager import Manager
-    from datetime import timedelta
 
 
 class ProgressManager:
