diff --git a/pyproject.toml b/pyproject.toml
index 7a310b22c1a8..a7814f60ef3e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,9 +12,9 @@ include = ["CHANGELOG.md"]
 [tool.poetry.dependencies]
 python = ">=3.11,<3.13"
 aiofiles = "^24.1.0"
-aiohttp = "^3.11.10"
-aiolimiter = "^1.2.0"
-aiosqlite = "0.17.0"
+aiohttp = "^3.10.10"
+aiolimiter = "^1.1.0"
+aiosqlite = "0.20.0"
 apprise = "^1.9.0"
 arrow = "^1.3.0"
 asyncpraw = "^7.7.1"
@@ -29,10 +29,10 @@ mediafire = "^0.6.1"
 myjdapi = "^1.1.7"
 pillow = "^11.0.0"
 platformdirs = "^4.3.6"
-pycryptodomex = "^3.21.0"
-pydantic = "^2.10.2"
+pycryptodomex = "^3.20.0"
+pydantic = "^2.9.2"
 pyyaml = "^6.0.2"
-rich = "^13.9.4"
+rich = "^13.8.1"
 send2trash = "^1.8.3"
 xxhash = "^3.5.0"
 
